benchmarks/multiplatform/build.gradle.kts

```diff
@@ -21,26 +23,21 @@ kotlin {
         binaries {
             executable {
                 entryPoint = "main"
-                freeCompilerArgs += listOf(
-                    "-linker-option", "-framework", "-linker-option", "Metal"
-                )
             }
         }
     }
     macosArm64 {
         binaries {
             executable {
                 entryPoint = "main"
-                freeCompilerArgs += listOf(
-                    "-linker-option", "-framework", "-linker-option", "Metal"
-                )
             }
         }
     }
 
     wasmJs {
-        browser()
         binaries.executable()
+        browser {
```

Comment:
Was something intended to be added there?

Reply:
Yes, it was. Fixed  :)